src/main/java/eu/xenit/gradle/docker/tasks/internal/NoImagesConfiguredException.java

```diff
@@ -0,0 +1,9 @@
+package eu.xenit.gradle.docker.tasks.internal;
+
+import org.gradle.api.GradleException;
+
+public class NoImagesConfiguredException extends GradleException {
+    public NoImagesConfiguredException() {
+        super("Pushing no docker images is not a valid action. Please configure some images to push.");
```

Comment:
It's a bit silly that this exception `NoImagesConfigured` has a hardcoded message about pushing images. But that can be extended later, if the need arises